### PR TITLE
📝 Storybook landing page

### LINKED ISSUE #1865: Redesign Storybook welcome page
Since the Storybook will get a more prominent role in the documentation (meaning we’ll create links from the developer tabs in Zeroheight to Storybook), we need to ensure a consistent look between the storybook and the storefront to the extent possible. One thing we can do is to use the same illustration that we use in the storefront, but replace the title with “EDS Core React”, with the expectation that there will be more storybooks as we expand to other technologies such as for example web components.

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1865

### CODE REVIEW

File: packages/eds-core-react/stories/docs/Intro.stories.mdx [hunk 1]
```diff
@@ -1,25 +1,45 @@
...
-<Typography variant="h1">Equinor Design System Storybook</Typography>
-<Typography variant="ingress">
-  This is the storybook for the packages of the Equinor Design
-  System. We have a <Typography link href="https://eds.equinor.com/">storefront</Typography> with
-  extensive documentation.
+# Equinor Design System Storybook
+
+<Typography variant="ingress"  style={{marginTop:'24px'}}>
+  This is the storybook for the <Typography link href="https://reactjs.org/">React</Typography> implementation of the Equinor Design
+  System. Our goal is to create and provided building blocks & resources for web development in Equinor.   We have a <Typography link href="https://eds.equinor.com/">storefront</Typography> with
```
Comment (martalalik) [inline packages/eds-core-react/stories/docs/Intro.stories.mdx:11]: Typo?: shouldn't be written `provide` instead `provided`?

File: packages/eds-core-react/stories/docs/Intro.stories.mdx [hunk 2]
```diff
@@ -1,25 +1,45 @@
...
+  extensive documentation for components, guidelines and patterns.
```
Comment (martalalik) [inline packages/eds-core-react/stories/docs/Intro.stories.mdx:12]: Should we mention `Figma` with link/links too?

File: packages/eds-core-react/stories/docs/Intro.stories.mdx [hunk 3]
```diff
@@ -1,25 +1,45 @@
...
+
+[![icons](https://img.shields.io/npm/v/@equinor/eds-icons?color=%23c3f3d2&label=%40equinor%2Feds-icons&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-icons)
+
+[![utils](https://img.shields.io/npm/v/@equinor/eds-utils?color=%23c3f3d2&label=%40equinor%2Feds-utils&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-utils)
+
+
+<Typography variant="h2" style={{marginTop:"32px"}}>Contact us</Typography>
+<Typography variant="caption" >
+  If you want to get in touch with the team, you’ll find us in the Equinor slack channel  <Typography link href="https://equinor.slack.com/archives/CJT20H1B9">#eds-design-system</Typography>
+</Typography>
```
Comment (martalalik) [inline packages/eds-core-react/stories/docs/Intro.stories.mdx:45]: `Get in tuch with us...` The text could be more encouraging, engaging for the contact - of course if we want that 🤓

File: packages/eds-core-react/stories/docs/Intro.stories.mdx (hunk 1 above)
Comment (mimarz) [reply packages/eds-core-react/stories/docs/Intro.stories.mdx]: fixed

File: packages/eds-core-react/stories/docs/Intro.stories.mdx (hunk 2 above)
Comment (mimarz) [reply packages/eds-core-react/stories/docs/Intro.stories.mdx]: Maybe 🤔 I had it initially, but then removed it as we have links to the Figma components on each component. 

I also changed this to be more of a guidelines card. So have another look :)

File: packages/eds-core-react/stories/docs/Intro.stories.mdx (hunk 3 above)
Comment (mimarz) [reply packages/eds-core-react/stories/docs/Intro.stories.mdx]: Changed wording and fixed typo

File: packages/eds-core-react/stories/docs/Intro.stories.mdx [hunk 4]
```diff
@@ -1,25 +1,65 @@
...
+
+[![lab](https://img.shields.io/npm/v/@equinor/eds-lab-react?color=%23c3f3d2&label=%40equinor%2Feds-lab-react&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-lab-react)
+
+
+[![utils](https://img.shields.io/npm/v/@equinor/eds-utils?color=%23c3f3d2&label=%40equinor%2Feds-utils&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-utils)
 
-[![Github](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com/equinor/design-system)
 
+## Contact us
+Get in touch with the EDS team and community on our Equinor Slack channel [#eds-design-system<](https://equinor.slack.com/archives/CJT20H1B9">)
```
Comment (martalalik) [inline packages/eds-core-react/stories/docs/Intro.stories.mdx:65]: Typo: `>` at the end of the link

**Comment by martalalik** [comment]
Looks awesome with the cards 👏

File: packages/eds-core-react/stories/docs/Intro.stories.mdx (hunk 4 above)
Comment (mimarz) [reply packages/eds-core-react/stories/docs/Intro.stories.mdx]: Fixed!

**Review by martalalik — APPROVED** [review]
LGTM 👍

File: packages/eds-core-react/.storybook/components/InfoCard.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+} = tokens
+
+type InfoCardTypes = {
+  title?: string
+  description?: string
+  url?: string
+  urlTitle?: string
+}
+
+const SBCard = styled(Card)`
```
Comment (oddvernes) [inline packages/eds-core-react/.storybook/components/InfoCard.tsx:20]: Maybe wait for the card changes and see if some stuff can be removed from here then

Comment (mimarz) [reply packages/eds-core-react/.storybook/components/InfoCard.tsx:20]: Update with latest changes to `Card`

File: packages/eds-core-react/stories/docs/Intro.stories.mdx [hunk 6]
```diff
@@ -1,25 +1,65 @@
...
+
+[![lab](https://img.shields.io/npm/v/@equinor/eds-lab-react?color=%23c3f3d2&label=%40equinor%2Feds-lab-react&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-lab-react)
+
+
+[![utils](https://img.shields.io/npm/v/@equinor/eds-utils?color=%23c3f3d2&label=%40equinor%2Feds-utils&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-utils)
 
-[![Github](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com/equinor/design-system)
 
+## Contact us
+Get in touch with the EDS team and community on our Equinor Slack channel [#eds-design-system<](https://equinor.slack.com/archives/CJT20H1B9)
```
Comment (oddvernes) [inline packages/eds-core-react/stories/docs/Intro.stories.mdx:65]: There is a "< "here #eds-design-system<

Comment (mimarz) [reply packages/eds-core-react/stories/docs/Intro.stories.mdx]: Fixed

**Pull request merged**